routes/authRoutes.js

```diff
@@ -86,6 +93,12 @@ router.route('/twitter')
 router.route('/twitter/redirect')
```

Comment:
Based on this story, is this still useful?

Reply:
No its not. I will remove it right away

Reply:
No. removing it right away